<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionvariableWindow</class>
 <widget class="QWidget" name="gestionvariableWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="button_quit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="vueVariable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>371</width>
     <height>231</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Identificateur</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Litt√©rale</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="button_refresh">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Rafraichir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
